core/src/test/java/org/elasticsearch/common/network/NetworkModuleTests.java

```diff
@@ -163,32 +160,6 @@ public void testRegisterHttpTransport() {
         assertFalse(module.isTransportClient());
     }
 
-    public void testRegisterRestHandler() {
```

Comment:
I think we should still have the equivalent of this test? See for example what I added in `IndicesModuleTests` for `MapperPlugin`.